pkg/mutate/patch.go

```diff
@@ -94,9 +100,24 @@ func (sc *SidecarConfig) ConfigFromAnnotations(annotations map[string]string) {
 			}
 		}
 		sc.Containers[i].Args = append(sc.Containers[i].Args, fmt.Sprintf("--%s=%s", monitoringOptsArg, strings.Join(opts, " ")))
+		// sc.Containers[i].Args = append(sc.Containers[i].Args, "--verbose=debug")
```

Comment:
Looks this string is unneeded, please remove.

Reply:
done